tools/build_defs/framework.bzl

```diff
@@ -249,8 +249,9 @@ def _check_file_name(var, name):
 
   if (not var[0:1].isalpha()):
     fail("{} should start with a letter.".format(name.capitalize()))
-  if (not var.isalnum()):
-    fail("{} should be alphanumeric.".format(name.capitalize()))
+  for letter in var:
```

Comment:
iterable strings are going away in Skylark, please use a for loop over indices

Reply:
Corrected

Reply:
Yep, actually it was reformatted by the plugin (sorry now I checked and see there is a setting how to turn it off)